src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$BodyFont: arial, helvetica, sans-serif;
+$FontSize: 30px;
+$daySize: 100px;
```

Comment:
Use the same naming for every variable. It's better to use `camelCase` everywhere. Don't use different approaches.